src/csharp/Microsoft.Spark.Extensions.Delta/Microsoft.Spark.Extensions.Delta.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@rapoth the package version should be follow the delta package version, not Spark .NET version right?